libs/ui-lib-tests/cypress/integration/static-ip/1-configure-static-ip-network-wide.cy.ts

```diff
@@ -107,6 +107,22 @@ describe(`Assisted Installer Static IP Network wide Configuration`, () => {
         .getWizardStepNav('Host specific configurations')
         .should('have.class', ACTIVE_NAV_ITEM_CLASS);
     });
+
+    it('nrt Can use IPv4 and IPv6 on DNS when dual stack', () => {
+      staticIpPage.dualStackNetworking().click();
+      [
+        '192.168.1.45',
+        '8c86:31b3:d1f9:f62c:5fbd:c152:147f:eaf8',
+        '192.168.1.46,cf72:7c8c:484d:033d:37d1:1696:8f89:632f',
+        'c727:f720:e48e:ea29:bf92:c777:488e:5d55,192.168.1.45',
+        '192.168.1.47,192.168.1.48',
+        'db9c:b59a:ca26:fe30:cdeb:39d2:19b1:aa68,a6c4:b658:a2b5:f590:3b00:1506:56aa:519c',
+      ].forEach((dnsEntry) => {
```

Comment:
Maybe this list of addresses could be used later in another test, so why don 't we make it a fixture?
PTAL, at https://docs.cypress.io/api/commands/fixture
You can also just `import` the fixture.

```suggestion
      cy
        .fixture('data/ipv4-and-ipv6-addresses.json')
        .then((addresses) => addresses.forEach((dnsEntry) => {
          // ...
        });
```